web/api/rest-api-design/Batching-Filtering-Pagination.md

```diff
@@ -0,0 +1,73 @@
+## Batching, Filtering, and Pagination
+
+### Filtering:
+
+- Topics covered: Elasticsearch and URL Parameters
```

Comment:
The way this is organized, it makes it sound like Elasticsearch and URL parameters are two different ways to perform filtering. As far as I understand it, they are not analogous. 

URL parameters are how you define the way people will use your API to search. They are independent of how the query is interpreted in the backend. You should be able to swap out the backend and have the URL parameters continue to work.

Elasticsearch and Postgres are two server-side tools than can look up the data requested via the URL parameters.